src/pages/admin/AdminUsersPage.tsx

```diff
@@ -53,7 +53,11 @@ const AdminUsersPage: FC = () => {
             <CircularProgress />
           </StyledProgressWrapper>
```

Comment:
loading burde flyttes opp, slik at den vises kun en spinner når man loader, og den burde bare vises for første lasting slik at vi ikke "mister" all data når man fjerner en rolle (spesielt irriterende om man har paginering og er på side 2+)

Reply:
Det var ikke helt rett frem, så vi får ta det som en egen sak senere hvis det blir et problem